azure-pipelines.yml

```diff
@@ -84,8 +84,8 @@ stages:
           inputs:
             versionSpec: $(NodeJSVersion)        
 
-        - task: UseDotNet@2
-          displayName: Add dotnet 3.0
+        - task: UseDotNet@5
```

Comment:
Why not call this something like `AddDotNetSDK` so the naming is more durable?

Reply:
sounds ok

Reply:
The task `UseDotNet@2` should stay the same.  `UseDotNet` is the name of an Azure DevOps thing that we don't control and `@2` is their plugin/action version that we also don't control.  The only thing you can and should change is the `displayName` field.

Reply:
done